apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/Trace.cs

```diff
@@ -55,15 +57,15 @@ namespace Google.Devtools.AspNet
     /// 
     /// Docs: https://cloud.google.com/trace/docs/
     /// </remarks>
-    public sealed class CloudTrace
+    public sealed class Trace
```

Comment:
Ah, here's the conflict... given that we've already got a namespace called Google.Cloud.Trace.V1 and a Trace class within that, adding *another* class called Trace feels like it's causing needless confusion. Can we just keep the CloudTrace name, avoiding all conflicts?